platform/stm32f7xx/qspi.c

```diff
@@ -375,7 +377,16 @@ static ssize_t spiflash_bdev_erase(struct bdev* device, off_t offset,
 
 static int spiflash_ioctl(struct bdev* device, int request, void* argp)
 {
-    return ERR_NOT_IMPLEMENTED;
+    int ret = ERR_NOT_SUPPORTED;
+
+    switch (request) {
+        case BIO_IOCTL_GET_MEM_MAP:
```

Comment:
There's a PUT_MEM_MAP too, should support that for when writes are needed. Unless it supports writing while in linear mode?